studio-server/pom.xml

```diff
@@ -104,6 +104,11 @@
       <artifactId>coremedia-common</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.coremedia.cms</groupId>
```

Comment:
why is this needed now?

Reply:
mvn told me.
update of the dependency plugin version?
or 2110.2 vs 2110.3?